### ISSUE TITLE
Installation fails on windows10

### ISSUE THREAD

**Issue by CsnCaio** [Requarks/wiki#579]
### Actual behavior
[1/6] Fetching 7zip helper... OK
[2/6] Fetching latest build... OK
[3/6] Fetching dependencies... OK
[4/6] Extracting app files...

Command Line Error:
Unknown switch:
-
Error: 7-Zip command line error
No linha:47 caractere:9
+     7 { throw "Error: 7-Zip command line error" }
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (Error: 7-Zip command line error:String) [], RuntimeException
    + FullyQualifiedErrorId : Error: 7-Zip command line error

### Expected behavior
Successfully installation.

### Steps to reproduce the behavior
- Open PowerShell as Administrator
- Set-ExecutionPolicy Bypass (then digit S (pt-br = Sim = Yes))
- iex ((New-Object System.Net.WebClient).DownloadString('https://wiki.js.org/install.ps1'))

Please, help me.

**Comment by NGPixel** [comment]
I was able to install Wiki.js on Windows 10 x64 build 1803 without any issues:

![image](https://user-images.githubusercontent.com/15522395/42718575-1f51a3e8-86d8-11e8-9c86-e78318049bca.png)

Make sure you have the permissions to write to the directory you specified. You can also try to run the install command on a non-elevated powershell prompt.

**Comment by PureNukage** [comment]
Same issue. Windows 10, x64, stops after Step 4 with a 7-Zip error. 

![image](https://user-images.githubusercontent.com/31448973/50982407-c1413d80-14c2-11e9-9f1e-9f908eabf55a.png)


I am an administrator, up-to-date with Git, node.js and MongoDB. 

Inside my wikijs folder I have '7za.exe', 'node_modules.tar.gz' and 'wiki-js.tar.gz'

**Comment by PureNukage** [comment]
and before it gets to this point, it displays: 

![image](https://user-images.githubusercontent.com/31448973/50982652-488eb100-14c3-11e9-9630-aae82112ce7f.png)


When I was following the Instructions on the website, the 1 line of Powershell code does not work, it comes up with a TLS error. 

![image](https://user-images.githubusercontent.com/31448973/50982715-6cea8d80-14c3-11e9-95ef-86781549d038.png)


This has been one of the most frustrating attempts to get something to work.

**Comment by PureNukage** [comment]
I have moved passed this. 

I followed the manual installation instructions found here: https://docs-legacy.requarks.io/wiki/install

I will open a new issue with the new issue I am encountering.
